### ISSUE TITLE
Does the new mongo cli require eval?

### ISSUE THREAD

**Issue by arichiardi** [krisajenkins/ob-mongo#8]
Hi there!

This is a very cool package thanks!

I have a weird problem here and it might be due to the `mongo` cli behavior.

I basically receive "Code block produced no output".

And if I go in debug mode, I see that the command that works in the terminal is different from the one executed here:

```
docker exec mongo_1 mongo --verbose db-name --eval  'coll = db.getCollection("subject"); coll.find().pretty()'
```

The above actually succeeds and prints out.

It turns out the missing part is `--eval`.

Now, I am not sure if I should send a patch, wanted to confirm your behavior first.

**Comment by gamecubate** [comment]
This is such a good idea ... storing queries in code blocks ... Kudos. But same here ... code block evaluation fails with same output.

**Comment by thisago** [comment]
Yes, indeed. After some tests I got it to work better by writing a temporary file instead, it seems that `--eval` works only when passing the code as argument, not piping (which it is what `org-babel-eval` does). I made a working package based on this: https://github.com/thisago/ob-mongosh

Thanks for the inspiration
